pkgs/by-name/hy/hypnotix/package.nix

```
@@ -15,13 +15,13 @@
 
 stdenv.mkDerivation rec {
   pname = "hypnotix";
-  version = "5.2";
+  version = "5.3";
 
   src = fetchFromGitHub {
     owner = "linuxmint";
     repo = "hypnotix";
     rev = version;
```

```suggestion
    tag = version;
```
and use `src.tag` in the changelog